core/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Mobilecoin core library.
+//! This provides base types and common functions for mobilecoin implementers /
+//! consumers.
+
+#![no_std]
+#![warn(missing_docs)]
+#![deny(unsafe_code)]
+#![allow(non_snake_case)]
+
+#[cfg(feature = "alloc")]
```

Comment:
because i'm extracting things from my ~~fanfic~~ fork there's a bit of cludge around like this, there shouldn't be an `alloc` feature at the moment but in my alternate universe there is. good catch / sorry for the volume of these!